src/components/JournalPage/index.jsx

```diff
@@ -37,8 +38,8 @@ class JournalPage extends React.Component {
                   // return <XBlockVideo />
                   return <div>XBlockVideo</div>;
                 case IMAGE:
-                  // return <Image />
-                  return <div>Image</div>;
+                  // TODO: get the right title and caption
```

Comment:
the title exists already and can be used. If you are not seeing it check that the page query is using fields=*, for example:
`http://localhost:18606/api/v1/content/pages/?type=journals.JournalPage&fields=*`
should return title in the values dict for the image

Reply:
also for now please default alt-text to use the title until we have a proper alt-text field on the model

Reply:
I thought that was the file title of the image, just to be used for authoring not for the end user to view?  Also title in the authoring tool is a required field and I assume we don't want to force authors to title every image because it may not always make sense to.

Reply:
for now we only have one title, which is specified on the image. eventually we'll expose a title override and caption override on the page where you insert the image but we don't have that currently

Reply:
okay sounds good i'll have the current title fill in for title and alt-text